docs/section11/Rule11-8.md

```diff
@@ -0,0 +1,54 @@
+# Service_Water_Heating - Rule 11-8
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-8
+
+**Rule Description:** "One system per building area type shall be modeled in the baseline."
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b
+
+**Evaluation Context:** B-RMD each SHW type
+**Data Lookup:**   
+**Function Call:** 
+- **get_SHW_types_and_spaces**
+- **get_SHW_equipment_connected_to_use_type**  
+
+**Applicability Checks:**
+- check that the SHW type in the P_RMD has SHW loads - **is P_RMD appropriate for the applicability check, or should it be B_RMD?**  
```

Comment:
Recommend avoiding "SHW type" language as it is not clear what it means.

Reply:
To your question, I think checking both makes sense. Although, maybe it is better to check the B_RMD in case there is an allowed measure in the proposed that eliminate loads.

Reply:
To your question, I Don't feel strongly but generally I think you woudl check the proposed in instances like this.

Reply:
discussed on call 6/5/2024 - OK to keep reference to proposed